tests/pipeline/test_all.nf.test

```diff
@@ -17,34 +17,42 @@ nextflow_pipeline {
             assertAll(
                 { assert workflow.success },
                 { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
-                { assert new File("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml").exists() },
+                { assert new File("$outputDir/pipeline_info/nf_core_metapep_software_mqc_versions.yml").exists() },
                 { assert snapshot(path("$outputDir/db_tables/alleles.tsv"),
                                 path("$outputDir/db_tables/conditions_alleles.tsv"),
                                 path("$outputDir/db_tables/conditions.tsv"),
                                 path("$outputDir/db_tables/entities_proteins.tsv"),
```

Comment:
Shouldn't here `microbiomes_entities.tsv` be added (because the entities, i.e. taxa, themselves do not change), while `entities_proteins.tsv` would be expected to change? That's what we listed in our meeting, or is this not what you observed in your tests?

Reply:
`microbiomes_entities.tsv` has a different md5sum compared to the last release. It is not clear why this happens. 
I will put `entities_proteins.tsv` to the list of files to be only checked for existence.